src/utils/contract/contract.ts

```diff
@@ -9,7 +9,9 @@ import { LAMPORTS_PER_SOL, PublicKey } from '@solana/web3.js';
 import { type EarnReloaded, IDL } from './program';
 
 const PROGRAM_ID = process.env.NEXT_PUBLIC_PAYMENT_PROGRAM_ID || '';
-const RPC_URL = process.env.NEXT_PUBLIC_PAYMENT_RPC_URL || '';
+const RPC_URL =
```

Comment:
why would you change this file?
please revert changes in this file and pull latest code from main

Reply:
Due to this change only my dasboard/listings seems to be working . It was  how i debugged the error. But i will change the required code.